### DISCUSSION TITLE
AI feature availability in Windows Client

### DISCUSSION THREAD

**Discussion by ano010** [SafeExamBrowser/seb-win-refactoring#297] [General]
Hi,

I have been working on integrating the Zoom/Jitsi service with SEB for live proctoring. It was successful with the help and support from the SEB community. AI feature supports with the IoS client but it does not for Windows client. I am curious to know whether this feature is available in the Windows SEB client (3.3.1).

Thanks

**Comment by dbuechel** [comment]
Unfortunately not, there is no AI functionality for the proctoring in SEB for Windows (and there most likely won't be in the near future).

**Reply by ano010** [reply]
Thanks for the prompt reply @dbuechel.
